resources/views/stores/show.blade.php

```diff
@@ -168,28 +165,28 @@
                     <div class="average">
                         <span class="star">
                             <span>
-                                {{$review->review > 1 ? '★' : '☆' }}
+                                {{$review->review >= 1 ? '★' : '☆' }}
                             </span>
                             <span>
-                                {{$review->review > 2 ? '★' : '☆' }}
+                                {{$review->review >= 2 ? '★' : '☆' }}
                             </span>
                             <span>
-                                {{$review->review > 3 ? '★' : '☆' }}
+                                {{$review->review >= 3 ? '★' : '☆' }}
                             </span>
                             <span>
-                                {{$review->review > 4 ? '★' : '☆' }}
+                                {{$review->review >= 4 ? '★' : '☆' }}
                             </span>
                             <span>
-                                {{$review->review > 5 ? '★' : '☆' }}
+                                {{$review->review >= 5 ? '★' : '☆' }}
                             </span>
                         </span>
-                        <span class="value">{{$review->review}}</span>
+                        <span class="value">{{$review->comment}}</span>
                     </div>
                 </div>
                 @if ($review->post_image !== null)
-                <img src="{{$review->post_image}}" alt="投稿写真">
+                <img src="{{$review->store->post_image}}" alt="投稿写真">
                 @else
```

Comment:
```suggestion
                @if ($review->post_image != null)
                <img src="{{$review->post_image}}" alt="投稿写真">
                @else
```
で条件文を変えたら出てきました。nullとの比較だからデータ型とかが関係なくてもいいのかなと思います。
口コミのところなのでstoreは必要なさそうです。